src/components/DocumentationTopic.vue

```diff
@@ -426,6 +430,15 @@ export default {
       || (downloadNotAvailableSummary && downloadNotAvailableSummary.length)
       || (primaryContentSections && primaryContentSections.length)
     ),
+    viewMoreLink: ({
+      interfaceLanguage,
+      objcPath,
+      swiftPath,
+      normalizePath,
+    }) => (
+      buildUrl(normalizePath(interfaceLanguage === Language.objectiveC.key.api
+        ? objcPath : swiftPath), { language: interfaceLanguage })
```

Comment:
Are we ok with including the language query parameter for swift? The code is a little cleaner this way since we don't have to conditionally set it to null just for swift.

Reply:
I think we should only include the param for ObjC since that's the only special with the default being Swift for pages available in both. Essentially, this link should work very similarly to the language toggle (except that it's for the current page, not the other language variant)